docs/source/using/basic-usage.adoc

```diff
@@ -258,8 +272,8 @@ $ minishift config unset memory
 === Driver-Specific Environment Variables
 
 You can also set driver-specific environment variables.
-Each docker-machine driver supports its own set of options and variables.
-A good starting point is the official docker-machine link:https://docs.docker.com/machine/drivers/[driver documentation].
+Each Docker Machine driver supports its own set of options and variables.
+A good starting point is the official Docker Machine link:https://docs.docker.com/machine/drivers/[driver documentation].
 
```

Comment:
@robin-owen I remember we had some discussion with @thatdocslady and agreed to use `docker` and not `Docker`. I do not remember the details. Are you changing these as per some guideline?

Reply:
Yes, following the [Docker Trademark Guidelines](https://www.docker.com/trademark-guidelines). It is explicitly mentioned here that we refer to Docker (and related marks) with proper capitalization unless we are referring to the `docker` command.